packages/shared/locales/en.json

```
@@ -133,15 +136,52 @@
             "body1": "This is an automated process to transfer your funds to your new wallet. We will check for spent addresses that weaken your security.",
             "body2": "It may take some time. Do not turn off your device during the migration.",
             "existing": "Your old wallet",
-            "new": "Your new wallet"
+            "new": "Your new wallet",
+            "learn": "Learn about the migration",
+            "beginMigration": "Begin Migration",
+            "migrating": "Migrating funds...",
+            "migrated": "{balance} migrated",
+            "migrationFailed": "Migration Failed"
+        },
+        "bundleMiningWarning": {
+            "title": "Warning",
+            "body1": "You have funds detected on a spent address. ",
+            "body2": "These are at risk of theft.",
+            "body3": "We recommend you run an automated process to help secure and transfer your funds safely.",
+            "learn": "Learn about spent addresses"
+        },
+        "secureSpentAddresses": {
+            "title": "Secure spent addresses",
+            "body1": "You have funds detected on {number} spent address.",            
```

Since this can be plural, we should use ICU syntax

```suggestion
            "body1": "You have funds detected on {number, plural, one {# spent address} other {# spent addresses}}.", 
```